demo/app.js

```diff
@@ -21,28 +29,46 @@ class App extends React.Component {
       mediaUrl: null,
       isTextEditable: true,
       spellCheck: false,
-      sttType: 'bbckaldi',
+      sttType: "bbckaldi",
       analyticsEvents: [],
-      title: '',
-      fileName: ''
+      title: "",
+      fileName: "",
+      autoSaveData: {},
+      autoSaveFormat: "draftjs",
```

Comment:
in the actual `TranscriptEditor` component is treated like props. Here in demo I added it in state so that I could display it here [demo/app.js#339](https://github.com/bbc/react-transcript-editor/pull/201/files#diff-b705f9ad2d951e512083753f54b4cad2R339) as well as if we want to add an option in the demo navbar to change that setting and see different on save formats etc..